homeassistant/components/tplink_omada/controller.py

```diff
@@ -16,15 +18,40 @@ class OmadaSiteController:
     """Controller for the Omada SDN site."""
 
     _gateway_coordinator: OmadaGatewayCoordinator | None = None
-    _initialized_gateway_coordinator = False
-    _clients_coordinator: OmadaClientsCoordinator | None = None
-
-    def __init__(self, hass: HomeAssistant, omada_client: OmadaSiteClient) -> None:
+    _devices_coordinator: OmadaDevicesCoordinator
+    _clients_coordinator: OmadaClientsCoordinator
```

Comment:
Might as well just remove the underscore and make them public, then you can remove the property exposing them

Reply:
I don't have a strong feeling either way. I'm aiming to follow whatever pattern the HA developer community prefers. Happy to change it back. I realised I didn't actually need to define the fields and the properties in the previous version either. So maybe keeping the properties, but leaving the underscored named fields as implicit from the `__init__` method is a better way to go?

Reply:
Ok, I tidied it up some more while bringing back the read-only props. It's all learning👍 

Cheers! 🍻